app-admin/flannel/files/flanneld-rkt.service

```diff
@@ -27,10 +27,15 @@ ExecStart=/usr/bin/rkt run --net=host \
    --volume runsystemd,kind=host,source=/run/systemd,readOnly=false \
    --volume runflannel,kind=host,source=/run/flannel,readOnly=false \
    --volume ssl,kind=host,source=${ETCD_SSL_DIR},readOnly=true \
+   --volume certs,kind=host,source=/usr/share/ca-certificates,readOnly=true \
+   --volume resolv,kind=host,source=/etc/resolv.conf,readOnly=true \
    --mount volume=runsystemd,target=/run/systemd \
    --mount volume=runflannel,target=/run/flannel \
    --mount volume=ssl,target=${ETCD_SSL_DIR} \
+   --mount volume=certs,target=/etc/ssl/certs \
+   --mount volume=resolv,target=/etc/resolv.conf \
    ${FLANNEL_IMG}:${FLANNEL_VER} \
+   --exec /opt/bin/flanneld \
```

Comment:
I don't think we were. In the early-docker version we run it directly with /opt/bin/flanneld as the command (see below)

`ExecStart=/usr/libexec/sdnotify-proxy /run/flannel/sd.sock \
  /usr/bin/docker run --net=host --privileged=true --rm \
  --volume=/run/flannel:/run/flannel \
  --env=NOTIFY_SOCKET=/run/flannel/sd.sock \
  --env=AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID} \
  --env=AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY} \
  --env-file=${FLANNEL_ENV_FILE} \
  --volume=/usr/share/ca-certificates:/etc/ssl/certs:ro \
  --volume=${ETCD_SSL_DIR}:${ETCD_SSL_DIR}:ro \
${FLANNEL_IMG}:${FLANNEL_VER} /opt/bin/flanneld --ip-masq=true`